docs/docs/llms/MLflow Tracing/introduction.mdx

```diff
@@ -0,0 +1,1146 @@
+---
+description: MLflow Tracing is a feature that enables LLM observability in your apps. MLflow automatically logs traces for LangChain, LlamaIndex, and more.
+sidebar_position: 1
+---
+
+import { APILink } from "@site/src/components/APILink";
+import TOCInline from "@theme/TOCInline";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Introduction to MLflow Tracing
+
+:::note
+MLflow Tracing is currently in **Experimental Status** and is subject to change without deprecation warning or notification.
+:::
+
+<section>
+  <div class="logo-grid">
+    <a href="../langchain/autologging.html">
+      <div class="logo-card">
+        <img src="/images/logos/langchain-logo.png" alt="LangChain Logo" />
+      </div>
+    </a>
```

Comment:
for all images (and links where possible), let's use the markdown syntax, otherwise the image links break during build (see the [preview job](https://github.com/mlflow/mlflow/pull/13707#issuecomment-2462981952), this is an accurate reproduction of what the site will be like in production).

it seems docusaurus does not process JSX/HTML in the same way as markdown, so let's use markdown whenever possible!

```suggestion
    <div class="logo-card">
      [![LangChain Logo](/images/logos/langchain-logo.png)](../langchain/autologging.html)
    </div>
```